Description: Initial mythbuntu-control-panel package for PPA
 MCP helps minimize the need to use the command line to install
 and maintain MythTV on Ubuntu.
 .
 mythbuntu-control-panel (20200708-1) focal; urgency=medium
 .
   * Initial mythbuntu-control-panel package for PPA
Author: Ted L <ted1193-lp@yahoo.com>

--- /dev/null
+++ mythbuntu-control-panel-20200708/mythbuntu-control-panel.1
@@ -0,0 +1,32 @@
+.TH "MYTHBUNTU-CONTROL-PANEL" "1" "10 May 2020"
+
+.SH "NAME"
+mythbuntu\-control\-panel \- graphical mythbuntu configuration utility
+
+.SH "DESCRIPTION"
+\fBmythbuntu\-control\-panel\fP is a graphical application designed to ease
+the configuration MythTV related processes on Ubuntu systems.
+
+.SH "USAGE"
+mythbuntu\-control\-panel [\-h|\-\-help] [\-\-debug] [\-\-plugin\-root\-path PATH] [\-l|\-\-logfile FILE]
+
+\-h, \-\-help   Shows the application usage
+
+\-\-debug       Frontend debug mode.  Useful for plugin development when plugins
+                implement the logging class.
+                NOTE: this does NOT start the backend in debug mode.  If you
+                are doing plugin development, be sure to manually restart the
+                backend between different runs.  The backend also loads plugins
+                into memory so you may otherwise see inconsistencies.
+
+\-\-plugin\-root\-path PATH Defines where to look for plugins.  The default system
+                            path will be used if this directory is missing
+                            python/ and glade/ or this directory isn't specified.
+
+\-l FILE, \-\-logfile FILE  Specifies an alternative place to log output when
+                            the frontend is being ran in debug mode.
+
+.SH "AUTHOR"
+\fBmythbuntu\-control\-panel\fP is based on MCC, originally written by Mario
+Limonciello <superm1@mythbuntu.org> and is licensed under the GNU General
+Public License, version 2 or later.
